bzu-gmb-launcher.sh

```diff
@@ -17,42 +17,41 @@ else
 exit 0
 fi
 
-#проверка установлен или нет yad и другое необходимое ПО для bzu-gmb
-dpkg -s yad | grep installed > /dev/null || echo 'no installing yad :(' | echo "$pass_user" | sudo -S apt install -f -y yad
-YadStatus=`dpkg -s yad | grep installed`
-echo "YAD" $YadStatus
-
-#проверяем установлена утилита inxi - информация о низкоуровневом ПО и железе
-dpkg -s inxi | grep installed > /dev/null || echo 'no install inxi :(' | echo "$pass_user" | sudo -S apt install -f -y inxi
-inxistatus=`dpkg -s inxi | grep installed`
-echo "INXI" $inxistatus
-
-#проверяем установлена утилита meson - она необходима для сборки многих программ из исходников
-dpkg -s meson | grep installed > /dev/null || echo 'no install meson :(' | echo "$pass_user" | sudo -S apt install -f -y meson
-inxistatus=`dpkg -s inxi | grep installed`
-echo "meson" $inxistatus
-
-#проверяем установлена утилита ninja-build - она необходима для сборки многих программ из исходников
-dpkg -s ninja-build | grep installed > /dev/null || echo 'no install ninja-build :(' | echo "$pass_user" | sudo -S apt install -f -y ninja-build
-inxistatus=`dpkg -s ninja-build | grep installed`
-echo "ninja-build" $inxistatus
-
-#проверяем установлена утилита p7zip-rar - она необходима для установки многих программ
-dpkg -s p7zip-rar | grep installed > /dev/null || echo 'no install p7zip-rar :(' | echo "$pass_user" | sudo -S apt install -f -y p7zip-rar rar unrar unace arj
-inxistatus=`dpkg -s ninja-build | grep installed`
-echo "p7zip-rar" $inxistatus
-
-#проверяем установлена утилита python-tk - она необходима для установки многих программ
-dpkg -s python-tk | grep installed > /dev/null || echo 'no install p7zip-rar :(' | echo "$pass_user" | sudo -S apt install -f -y python-tk
-inxistatus=`dpkg -s python-tk | grep installed`;echo "python-tk" $inxistatus
-
-#проверяем установлена утилита xosd-bin - она необходима для работы многих программ
-dpkg -s xosd-bin | grep installed > /dev/null || echo 'no install xosd-bin :(' | echo "$pass_user" | sudo -S apt install -f -y xosd-bin
-inxistatus=`dpkg -s xosd-bin | grep installed`;echo "xosd-bin" $inxistatus
-
-#проверяем установлена утилита aptitude - она необходима для работы многих программ
-dpkg -s aptitude | grep installed > /dev/null || echo 'no install aptitude :(' | echo "$pass_user" | sudo -S apt install -f -y aptitude
-inxistatus=`dpkg -s aptitude | grep installed`;echo "aptitude" $inxistatus
+# проверка установлен ли пакет
+is_installed()
+{
+  dpkg -s $1 &> /dev/null
+}
```

Comment:
```suggestion
is_installed()
{
  dpkg -s $1 &> /dev/null
}
```

is_installed() -- объявление функции 
{ } -- начало и конец тела функции
$1 -- первый параметр функции, к примеру, при вызов ```is_installed yad``` будет вызван ```dpkg -s yad &> /dev/null```
&> перенаправить весь вывод, т.е. стандартный вывод и стандартный вывод ошибок будет перенаправлен в /dev/null

Результат вызова функции это exit code вызова dpkg
Если пакет есть, то exit code 0, иначе 1

Вызов ```is_installed yad && echo "Yad уже установлен" || install yad``` будет работать следующим образом:
Если yad установлен, то выведется сообщение "Yad уже установлен", выполнение строки завершается.
Если yad не установлен, то вызовется ```install yad```, выполнение строки завершается.